src/constants/common.ts

```diff
@@ -318,6 +318,8 @@ export const MODAL_RECIPIENT_INFO = 'recipient-info';
 export const MODAL_RESET_WALLET = 'reset-wallet';
 export const MODAL_TRANSFER_RECEIVE = 'transfer-receive';
 export const MODAL_TRANSFER_SEND = 'transfer-send';
+export const MODAL_BOWSER_ACTIONS_DAPP = 'browser-actions';
+export const MODAL_WARNING_DAPP_BROWSER = 'warning-dapp-browser';
```

Comment:
```suggestion
export const MODAL_DAPP_BROWSER_ACTIONS = 'browser-actions';
export const MODAL_DAPP_BROWSER_WARNING = 'warning-dapp-browser';
```
1. There is no such a thing as "bowser" (maybe besides Mario world) :)
2. When naming thing try to start from general and finish with specific.